nautobot/core/apps/__init__.py

```diff
@@ -691,6 +691,18 @@ def convert_biginteger(field, registry=None):  # pylint: disable=redefined-outer
 
         monkey_mix(TaggableManager, mixins.TaggableManagerMonkeyMixin)
 
+        from django.db.models.sql.query import Query
+
+        Query._set_values = Query.set_values
+
+        def set_values(self, fields):
+            if fields:
+                for field in fields:
+                    self.check_alias(field)
+            self._set_values(fields)
+
+        Query.set_values = set_values
```

Comment:
Much cleaner! 

Please add docstring and/or comments referencing the CVE, the upstream PR that fixed the CVE, and any other relevant context.